buildenv/jenkins/common/pipeline-functions

```diff
@@ -78,15 +78,44 @@ def get_sha(REPO, BRANCH) {
 
 def build(BUILD_JOB_NAME, OPENJDK_REPO, OPENJDK_BRANCH, OPENJDK_SHA, OPENJ9_REPO, OPENJ9_BRANCH, OPENJ9_SHA, OMR_REPO, OMR_BRANCH, OMR_SHA, VARIABLE_FILE) {
     stage ('Build') {
-        JOB = build job: BUILD_JOB_NAME, parameters: [string(name: 'OPENJDK_REPO', value: OPENJDK_REPO), string(name: 'OPENJDK_BRANCH', value: OPENJDK_BRANCH), string(name: 'OPENJDK_SHA', value: OPENJDK_SHA), string(name: 'OPENJ9_REPO', value: OPENJ9_REPO), string(name: 'OPENJ9_BRANCH', value: OPENJ9_BRANCH), string(name: 'OPENJ9_SHA', value: OPENJ9_SHA), string(name: 'OMR_REPO', value: OMR_REPO), string(name: 'OMR_BRANCH', value: OMR_BRANCH), string(name: 'OMR_SHA', value: OMR_SHA), string(name: 'VARIABLE_FILE', value: VARIABLE_FILE)]
-        return JOB
+        return build_with_slack(BUILD_JOB_NAME, [string(name: 'OPENJDK_REPO', value: OPENJDK_REPO), string(name: 'OPENJDK_BRANCH', value: OPENJDK_BRANCH), string(name: 'OPENJDK_SHA', value: OPENJDK_SHA), string(name: 'OPENJ9_REPO', value: OPENJ9_REPO), string(name: 'OPENJ9_BRANCH', value: OPENJ9_BRANCH), string(name: 'OPENJ9_SHA', value: OPENJ9_SHA), string(name: 'OMR_REPO', value: OMR_REPO), string(name: 'OMR_BRANCH', value: OMR_BRANCH), string(name: 'OMR_SHA', value: OMR_SHA), string(name: 'VARIABLE_FILE', value: VARIABLE_FILE)])
```

Comment:
Can these be formatted in a more readable way?  Maybe like:
```
return build_with_slack(BUILD_JOB_NAME, 
    [string(name: 'OPENJDK_REPO', value: OPENJDK_REPO), 
    string(name: 'OPENJDK_BRANCH', value: OPENJDK_BRANCH), 
    string(name: 'OPENJDK_SHA', value: OPENJDK_SHA), 
    string(name: 'OPENJ9_REPO', value: OPENJ9_REPO),    
    string(name: 'OPENJ9_BRANCH', value: OPENJ9_BRANCH), 
    string(name: 'OPENJ9_SHA', value: OPENJ9_SHA), 
    string(name: 'OMR_REPO', value: OMR_REPO), 
    string(name: 'OMR_BRANCH', value: OMR_BRANCH), 
    string(name: 'OMR_SHA', value: OMR_SHA), 
    string(name: 'VARIABLE_FILE', value: VARIABLE_FILE)])
```

Reply:
Shouldn't be a problem to change that